activesupport/test/message_verifier_test.rb

```diff
@@ -24,6 +24,7 @@ def test_valid_message
     data, hash = @verifier.generate(@data).split("--")
     assert !@verifier.valid_message?(nil)
     assert !@verifier.valid_message?("")
+    assert !@verifier.valid_message?("\xff")
```

Comment:
I think @matthewd's use of "throw" mislead you a bit. He just wants an `# invalid encoding` comment added to the line in the test :smile: